src/tests/integration/test-data/foundry-prove-all

```diff
@@ -145,9 +145,14 @@ LoopsTest.test_sum_1000()
 LoopsTest.testSumToN(uint256)
 LoopsTest.testSumToNBroken(uint256)
 MethodDisambiguateTest.test_method_call()
-MockCallTest.testMockCall()
+MockCallTest.testSelectorMockCall()
 MockCallTest.testMockCalls()
 MockCallTest.testMockCallValue()
+MockCallTestFoundry.testMockGetters()
+MockCallTestFoundry.testMockNested()
+MockCallTestFoundry.testMockSelector()
+MockCallTestFoundry.testMockCalldata()
+MockCallTestFoundry.testRevertMock()
```

Comment:
Similar to my previous comment. Could you time these on the legacy backend and add them to `foundry-prove-skip-legacy` if they take more than 300s?

Reply:
Yes! I'll do that.

Reply:
I confirm that all of them take less than 300s. I added one more test that also passes and takes less than 300s. I added some more tests from `MockCallTest.sol` but added them to the skip lists since they are not passing yet.